app/routes/flush.py

```diff
@@ -25,9 +29,11 @@
 
 logger = get_logger()
 
+Submitter = GCSSubmitter | LogSubmitter | RabbitMQSubmitter
```

Comment:
Worth using [TypeAlias](https://docs.python.org/3/library/typing.html#typing.TypeAlias)

Reply:
This led me down a really interesting hole reading about the [PEP 613](https://peps.python.org/pep-0613/) standard, nice one! Sorted now, added explicit hinting as a TypeAlias 👍